my-app/src/components/atoms/menuItemImage/MenuItemImage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './styles.css';
+
+function MenuItemImage(props) {
+  return (
+    <React.Fragment>
```

Comment:
`React.Fragment` is not required here.

As a follow up, read about when to use fragments.

Also, make this into a generic atom Image which can then be used as veg mark logo as well as menu item image depending on the props passed